rmw_opendds_cpp/src/rmw_init.cpp

```diff
@@ -111,4 +121,5 @@ rmw_context_fini(rmw_context_t * context)
   *context = rmw_get_zero_initialized_context();
```

Comment:
These lines are getting moved from `rmw_context_fini` to `rmw_shutdown`, so now except for the `RMW_CHECK_ARGUMENT_FOR_NULL` macro it looks like they're doing the same thing. Is that indented?

Reply:
@iguessthislldo was "indented" intended?

Reply:
Yes... I probably misspelled and went with whatever Chrome suggested.